### ISSUE TITLE
[Bug Report][2.6.3] Elements with role="dialog" must have way to get aria-label

### ISSUE THREAD

**Issue by awaigand** [vuetifyjs/vuetify#14755]
### Environment
**Vuetify Version:** 2.6.3
**Vue Version:** 2.6.11
**Browsers:** Firefox 97.0
**OS:** Windows 10

### Steps to reproduce
1. Use a v-dialog Element
2. Inspect the generated HTML

### Expected Behavior
One should be able to add aria-label, aria-labelledby and aria-describedby to the role="dialog" element in order to be accessible.

### Actual Behavior
It is not possible, as far as I can see.

### Reproduction Link
<a href="https://codepen.io/awaigand/pen/vYWjrLZ" target="_blank">https://codepen.io/awaigand/pen/vYWjrLZ</a>

### Other comments
Description of how to use role="dialog": https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role
WAI-ARIA practice: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

**Comment by lioneaglesolutions** [comment]
Closing as this is resolved in V3

https://play.vuetifyjs.com/#eNqNVEtu2zAQvcqAqxawJCAtUMBwChftskAPEGVBk+NkaopSyZGSIPDdOxQlV7ZRJFqR83nvzYe6e1Xfuq4celRrtWFsOqcZv9YeYGNpAON0jLe1YnzmwqBnDLUa3RIwFDv2+QxgWtcGiewCNTq81Co7pliAXx16+EHatQ/1KUkg7GTKBoChaFqLTpAmzwSUPm2YBs2ZRweRs/Q+keVH8eie27OsQLpwejeCMkYmv0A9CRzVGB3swnKyFakBZw6An23ABqiLfQM2VQ+RGHSDvJJu+IiGkfsA2lJH0QgroCNxRrSSAEh9lFohdV2SyRuyZHvP0DOIXoEH5AyN0OgHr0E7+tPr8kxh9T+Js/bUNtFzIT+P73pusHOtOcDWODKH0xjgFvbaRZTpf3dtxGmUiVxQzmlnQde8s+tkSoZMMC/VAnBTyQrKaVMtFnMTTaAuF4rP0jjpEO517xheM4TVrOHDx/kKENIY/L+7hIyU61zSarYf8+E4GuSyqSYutVKO/CGWv2Pr5aGMSLUyMdZqDXc5rVaPzF1cV5WxXiJliWkIpUeufNdU8sSY9i/bT+VNeSOFRZ5NaTdjOYIlpPvaH4WQ0lJw0ejugjU7RuapoloJUrq/IWErYVWQBaMGC9s2o5bPX7KYhb3E2BS70D5FDIJSq6lDI09S/A6ut8oViivoSvgGDIU8bIsh/WneV9JF2llZF76r0qZRH9VxpXKe9H4Ure7/Aj+doKE=

**Comment by deraw** [comment]
I think this should still be fixed in v2, so I've reopened my Pull Request

**Comment by michaelhastrich** [comment]
How is this fixed in V3? The `aria-label` set on the `v-dialog` in the playground example isn't exposed in the accessibility tree?
